docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -156,6 +156,7 @@ def commit_sha_suffix?(tag)
         # them (unless there's a `latest` version pushed to the registry, in
         # which case we'll use that to find the latest version)
         return false unless tag.match?(/(^|\-)[0-9a-f]{7,}$/)
+        return false if tag.match?(/(^|\-)\d+$/)
```

Comment:
Can you replace the `!tag.match?(/(^|\-)20[0-1]\d{5}$/)` line below with `!tag.match?(/(^|\-)\d+$/)` instead of this change, as otherwise it won't ever be reached.

Reply:
I've just updated it :)

Reply:
Awesome, I think we can remove this line then!

Reply:
good point! updated!